tests/mssql.yaml

```diff
@@ -19,22 +22,13 @@ spec:
       terminationGracePeriodSeconds: 10
       containers:
       - name: mssql
-        image: mcr.microsoft.com/mssql/server:2017-latest
+        #image: mcr.microsoft.com/mssql/server:2017-latest
+        image: quay.io/mkarg/mssql:latest
```

Comment:
Should this be in cloud-bulldozer's quay?

Reply:
I don't really know - it's the prepopulated mssql image. We can certainly shove it there, but it's only for the CI tasks. Your call.

Reply:
@jtaleric @aakarshg thoughts on where to put this?

Reply:
Do you've the dockerfile for this @mkarg75 ? if so, i think it'll be better to put this in cloud-bulldozer as if any changes are required in the future, we'll know where/what to make it?

Reply:
Nope, unfortunately not, I've pulled a mssql image, created the db and its contents in there and created the image from that. Since that's causing us issues on KNI right now, I might have to change that procedure anyway.

Reply:
okay, when you change the procedure please add a dockerfile for it so that anyone can repeat it as well.